spec/conformance/index.md

```diff
@@ -77,3 +108,166 @@ completing the launch.
+automated discovery of OAuth2 endpoints in its [CapabilityStatement]({{site.data.fhir.path}}/capabilitystatement.html) using the [OAuth Uri extension](#oauth-uris-extension) on the `rest.security` element (or, when using FHIR DSTU2, the
+`Conformance.rest.security` element). Any time a client sees this extension, it
+must be prepared to authorize using SMART's OAuth2-based protocol.
+
+The OAuth extension has the following internal components:
+
+|Component|Conformance Expectation|Type|Description|
+|---|---|---|---|
+|authorize|**SHALL**|`valueUri`|URL to the OAuth2 authorization endpoint.|
+|token|**CONDITIONAL SHALL**|`valueUri`|URL to the OAuth2 token endpoint, required unless the implicit grant flow is used.|
+|register|**SHOULD**|`valueUri`|If available, URL to the OAuth2 dynamic registration endpoint for this FHIR server.|
+|manage|**SHOULD**|`valueUri`|If available, URL where an end-user can view which applications currently have access to data and can make adjustments to these access rights.|
+|introspect|**SHOULD**|`valueUri`|URL to a server’s introspection endpoint that can be used to validate a token.|
+|revoke|**SHOULD**|`valueUri`|URL to a server’s endpoint that can be used to revoke a token.|
+{:.grid}
+
+
+<!-- =======  inline view of extension ============== -->
+
+#### OAuth Uris Extension
+
+Full StructureDefinition: [STU3]({{layout.baseurl}}/StructureDefinition-oauth-uris.html), [DSTU2]({{layout.baseurl}}/todo.html)
+
+{% include StructureDefinition-oauth-uris-inline.html %}
+
+<!-- =======    end inline view of extension ============== -->
+
+
+<!--
+### Publishing Additional Conformance Details
+
+A SMART on FHIR server should also describe which *optional* SMART core set of capabilities is it supports by declaring it within the CapabilityStatement [capabilities extension](#capabilities-extension) on the `rest.security` element (or, when using FHIR DSTU2, the
+`Conformance.rest.security` element).
+
+
+=======  inline view of extension ==============
+
+#### SMART Capabilities Extension
+
+Full StructureDefinition: [STU3](/StructureDefinition-extension-smart-capabilities.html), [DSTU2](/todo.html)
+
+{%raw%}{% include StructureDefinition-extension-smart-capabilities-inline.html %}{%endraw%}
+
+ =======    end inline view of extension ==============
+
+-->
+
+### Example
+
+{% include cs-example.md %}
+
+( for a complete example see the [CapabilityStatement Example](todo.html) )
+
+## FHIR Authorization Endpoint and Capabilities Discovery using a Well-Known Uniform Resource Identifiers (URIs)
+{: #using-well-known}
+
+As an alternative to using a FHIR CapabilityStatement, the authorization endpoints accepted by a FHIR resource server can be exposed as a Well-Known Uniform Resource Identifiers (URIs) [(RFC5785)][well-known] JSON document.
+
+FHIR endpoints requiring authorization MUST serve a JSON document at the location formed by appending `/.well-known/fhir-authorization-configuration` to their base URL.
```

Comment:
Our issue resolutions called this `smart-configuration` -- was the change intentional? I may just be missing something. (This appears in 8 places.)